examples/hotrod/README.md

```diff
@@ -44,3 +44,18 @@ Then open http://127.0.0.1:8080
 
 [hotrod-tutorial]: https://medium.com/@YuriShkuro/take-opentracing-for-a-hotrod-ride-f6e3141f7941
 [hotrod-openshift]: https://blog.openshift.com/openshift-commons-briefing-82-distributed-tracing-with-jaeger-prometheus-on-kubernetes/
+
+
+
+# HotROD on Containerized Version
```

Comment:
I would prefer if we didn't change the readme at this time. The first docker command won't run since the hotrod image is not actually published.

Reply:
Removed the changes README in d5250a6